slide.sh

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
+
+let CURRENTPAGE=0
+let TOTALSLIDES=$(grep "<<EOF" slideshow.sh | wc -l | tr -d " ")
```

Comment:
I actually don't think this is a safe assumption. Because slide.sh just takes input from STDIN, this could just as well be an echo-pipe (`echo | <something>`) and likely lots of other different ways of reading STDIN. This also brings in a dependency on `wc` and `tr`, which albeit are usually available, but still something to think about.